Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN apk add --no-cache \
   git \
   tini
 
+RUN npm install --global npm husky
```

Comment:
(Question, Non Important) Is this necessary to install a webhook library in a docker built image?

Reply:
to be transparent: It was marked as a missing dependency but I did not checked what it is. Shame on me 😅
WiP.

Reply:
Done in https://github.com/cicd-lectures/slides/pull/80. Husky is a dependency of NPM Check Updater `ncu`.